project.lua

```diff
@@ -0,0 +1,46 @@
+project "libtiff"
+	kind "StaticLib"
+	language "C++"
+	staticruntime "On"
+	tags { "vault-project" }
+
+	files 
+	{
+	  "project.lua",
+	  "libtiff/**.c",
+	  "libtiff/**.h" 
+	}
+	
+    excludes
+	{
+      "libtiff/mkg3states.c",
+      "libtiff/mkspans.c",
+      "libtiff/tif_acorn.c",
+      "libtiff/tif_atari.c",
+      "libtiff/tif_apple.c",
+      "libtiff/tif_msdos.c",
+      "libtiff/tif_unix.c",
+      "libtiff/tif_vms.c",
+      "libtiff/tif_wince.c",
+      "libtiff/tif_win3.c",
+      "libtiff/tif_win32.c"
+    }
+
+	includedirs {  }
+
+	-- Linking
+	links { }
```

Comment:
Just an FYI, in other static libs, the `links` are really just for decoration, we actually only use them to document the dependencies. Static libs don't actually run the linker - MSVC is an exception, and I believe it only does so sometimes.